data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/SimpleCrac.java

```diff
@@ -22,105 +22,206 @@
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.MINIMAL_CLASS)
 public class SimpleCrac extends AbstractIdentifiable implements Crac {
-    private List<Cnec> cnecs;
-    @JsonProperty("contingencies")
-    private List<Contingency> contingencies;
```

Comment:
Could you note somewhere in the java doc the advantages of using <Set> instead of <List> here?

Reply:
Could you note somewhere in the java doc the advantages of using "SET" instead of "LIST" here?